resources/[race]/race_sphud/sp_hud.lua

```diff
@@ -453,7 +453,12 @@ function drawIVHud (player, g_Veh, rank, players, currentCheckpoint, totalCheckp
 			dxDrawRectangle ( i, t, 1, h, tocolor(team[1] or 0, team[2] or 0, team[3] or 0,a) )
 		end
 		
-		dxDrawText ( "CHECKPOINT", l, t, m, b, textColor, labelScale, font, "right", "center", false, false, false, false, true)
+		if sx >= 800 and sy >= 600 then
```

Comment:
'>=' means 'equal to or higher than' so this change would only apply to resolutions lower than 800x600

Reply:
Ye, that was the point. First checked highers for doing the regular and using the var, and if it was lower use a custom value that should ajust the 'Checkpoint' getting out of the black background: https://mrgreengaming.com/forums/topic/20104-gui-being-dont-well-displayed/?tab=comments#comment-160235

Reply:
Ah, thanks for clarifying. I'll merge this and thanks for the help!